src/Terminal/HelpText.ts

```diff
@@ -215,6 +216,15 @@ export const HelpTexts: Record<string, string[]> = {
     "Download all text files: download *.txt",
     " ",
   ],
+  echo: [
+    "Usage: echo [string]",
+    " ",
+    "Print the specified string to the terminal. This command is mostly useful for piping",
+    " ",
+    "Example: echo 'Text To Store In File' > newFile.txt",
+    " ",
+    "Example: echo 'Text To Search In' | grep 'To'",
```

Comment:
```suggestion
    "Example: echo 'Text To Search In' | grep To",
```
Quoting is unnatural for a single word.